firstissue.json

```diff
@@ -1,1310 +1,1325 @@
-    "uber/kraken",
-    "ubuntu/microk8s",
-    "unisonweb/unison",
-    "UniversalMediaServer/UniversalMediaServer",
-    "unoplatform/uno",
-    "uutils/coreutils",
-    "valhalla/valhalla",
-    "validatorjs/validator.js",
-    "Varying-Vagrant-Vagrants/VVV",
-    "vectordotdev/vector",
-    "vercel/async-retry",
-    "vercel/commerce",
-    "vercel/examples",
-    "vercel/hazel",
-    "vercel/hyper",
-    "vercel/ncc",
-    "vercel/next-learn",
-    "vercel/next.js",
-    "vercel/nextjs-subscription-payments",
-    "vercel/og-image",
-    "vercel/pkg",
-    "vercel/platforms",
-    "vercel/satori",
-    "vercel/serve",
-    "vercel/styled-jsx",
-    "vercel/swr",
-    "vercel/turbo",
-    "vercel/vercel",
-    "vercel/virtual-event-starter-kit",
-    "vesoft-inc/nebula",
-    "vespa-engine/vespa",
-    "vicky002/AlgoWiki",
-    "VictoriaMetrics/VictoriaMetrics",
-    "vimeo/psalm",
-    "virtual-kubelet/virtual-kubelet",
-    "vitessio/vitess",
-    "vmware-tanzu/carvel-ytt",
-    "vmware-tanzu/community-edition",
-    "vmware-tanzu/octant",
-    "vmware-tanzu/sonobuoy",
-    "vmware-tanzu/velero",
-    "vmware/differential-datalog",
-    "vmware/govmomi",
-    "vmware/open-vm-tools",
-    "vmware/photon",
-    "vmware/pyvmomi",
-    "volcano-sh/volcano",
-    "volta-cli/volta",
-    "VSCodeVim/Vim",
-    "vuejs/vue",
-    "vuejs/vue-router",
-    "vuejs/vuepress",
-    "vuetifyjs/vuetify",
-    "VulcanJS/Vulcan",
-    "wagtail/wagtail",
-    "wasm3/wasm3",
-    "WasmEdge/WasmEdge",
-    "wasmerio/wasmer",
-    "wasp-lang/wasp",
-    "weaveworks/eksctl",
-    "weaveworks/ignite",
-    "weaveworks/weave",
-    "web-platform-tests/wpt",
-    "WebAssembly/spec",
-    "WebAssembly/WASI",
-    "webhintio/hint",
-    "webiny/webiny-js",
-    "WeblateOrg/weblate",
-    "wemake-services/wemake-python-styleguide",
-    "werf/werf",
-    "wesnoth/wesnoth",
-    "wger-project/wger",
-    "whatwg/fetch",
-    "whatwg/html",
-    "whitesmith/rubycritic",
-    "winstonjs/winston",
-    "winsw/winsw",
-    "woocommerce/woocommerce",
-    "woodpecker-ci/woodpecker",
-    "wordpress-mobile/WordPress-Android",
-    "wordpress-mobile/WordPress-iOS",
-    "WordPress/gutenberg",
-    "wp-graphql/wp-graphql",
-    "xenia-project/xenia",
-    "xtdb/xtdb",
-    "xtermjs/xterm.js",
-    "xunit/xunit",
-    "yarnpkg/yarn",
-    "Yelp/paasta",
-    "yewstack/yew",
-    "Yoast/wordpress-seo",
-    "yugabyte/yugabyte-db",
-    "zabbix/zabbix",
-    "zalando/skipper",
-    "zaproxy/zaproxy",
-    "zcash/zcash",
-    "zeebe-io/zeebe",
-    "zeek/zeek",
-    "zeit/hyper",
-    "zeit/next.js",
-    "zephyrproject-rtos/zephyr",
-    "zeromicro/go-zero",
-    "zio/zio",
-    "zitadel/zitadel",
-    "zsh-users/zsh-completions",
-    "zsh-users/zsh-syntax-highlighting",
-    "zulip/zulip"
-  ],
-  "labels": [
-    "good first issue",
-    "help wanted"
-  ]
-}
+[
+  {
+    "name": "GitLab",
+    "provider": "gitlab",
+    "repositories": [
+      "gitlab-org/gitlab|278964"
```

Comment:
@zillemarco Added some more repositories that I manually verified that have issues with one of the said labels, however it gives an error as fullName doesn't seem to be available for all of these.

Reply:
@taucher2003 are we close to having the namespace permission stuff fixed?

Essentially here we're having to query group.fullPath because namespace.fullPath doesn't work unless you're a member.

Reply:
@ltickett sadly not, the first step of creating the new ability that keeps the member behaviour is done, but probably the hardest part is the next one, as we need to identify and check all the points, where something like `:"read_#{object_type}"` is used in the ability checks. After that we should be good to move the namespace type authorization over to the one that works as intended with public namespaces.

Reply:
I added all those repos except `CalcProgrammer1/OpenRGB` to avoid issues with the group full name being blocked 🙂 

We can always add it later on 🙂

Reply:
Actually, `cznic/sqlite` and `timvisee/ffsend` had the same issue 😓